scss/_patterns_navigation.scss

```diff
@@ -919,4 +909,14 @@ $spv-navigation-logo-bottom-position: 0.125rem; // 2px when 1rem is 16px
       }
     }
   }
+
+  .p-navigation--sliding .p-navigation__item--dropdown-close {
```

Comment:
Is this just a source order changing? 

I'm asking because it's fine if that fixes this bug, but I'm a bit worried that we have issues like this (caused by source order), as this may mean there are other things that are affected by the order here.

Reply:
Yes, this is a source order issue - caused by the fact that our `.p-navigation__item--dropdown-close` elements themselves are nested within `p-navigation--sliding .p-navigation__item-dropdown-toggle .p-navigation__link` elements. 

Because the "Go Back" button is nested within the dropdown toggle `li` of the parent nav item, and the chevron style is applied to `p-navigation-link` inside of `p-navigation__item-dropdown-toggle`, the source order matters here. If the default style (the button to open a nested navigation) is after the style for the go-back button, then the go-back button's style is overridden, making the chevron point the wrong way.

I believe a source-order independent solution would require either one of the following:

- using the pre-existing `js-back` class in a style selector to exclude `p-navigation__link.js-back` from the style for the right-facing chevron 
- add some other class to the back-nav buttons to exclude them as with the previous option, but this would require users to change their HTML to add another class.

Reply:
@bartaz I made the requested change per our discussion to encapsulate all of the chevron-related styling into a single block:

![image](https://github.com/canonical/vanilla-framework/assets/46915153/25d96fa7-78f0-4f69-a0a4-a119055ae420)